server/models/admin.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const adminModel = (sequelize, DataTypes) => {
```

Comment:
Why have a separate model for admins? 
Normally you would use one model like Users, and add a flag for is_admin. 
But ok to keep this as-is for this app.